### PR TITLE
Development to master

### PR DESCRIPTION
**Pull request by cjrace**
**Minor changes:**

- Updating README
- Fixing typo in country code message
- Fixing breaking bug in invalid indicator unit tests (multiple failing units would have caused multiple messages to be printed, rather than a single message with multiple variables in it, breaking the tables that show messages in the UI)

**Main change:**
- Adding test data, unit test, and the test itself for flagging a warning if indicator dp isn't specified

The warning should appear as below. Feel free to suggest alternative wording.

image

### CODE REVIEW

File: R/mainTests.r [hunk 1]
```diff
@@ -1627,3 +1628,34 @@ indicator_dp_validation <- function(meta) {
...
+
+  if (length(blankIndicators) == 0) {
+    output <- list(
+      "message" = "The indicator_dp column is completed for all indicators.",
+      "result" = "PASS"
+    )
+  } else {
+    if (length(blankIndicators) == 1) {
+      output <- list(
+        "message" = paste0("The following indicator does not have a specified number of dp in the metadata file: '", paste(blankIndicators, collapse = "', '"), "', this should be explicity stated where possible."),
```
Comment (lauraselby) [inline R/mainTests.r]: Should it refer to indicator_dp specifically? Just wondering if analysts might miss where they're meant to add it.

File: README.md [hunk 2]
```diff
@@ -10,24 +27,67 @@ A shiny app to provider a user friendly way to QA data files using centralised c
...
 
 UI tests have been created using shinytest that test the app loads, that the reset button really does clear everything, and that files that we'd expect to hit each particular stage of the screening do so. More should be added over time as extra features are added.
 
 GitHub Actions provide CI by running the automated tests and checks for code styling, with badges in this README referring to the latest state of the master branch. The yaml files for these workflows can be found in the .github/workflows folder.
 
-## Project functions
+The function `run_tests_locally()` is created in the Rprofile script and is available in the RStudio console at all times to run both the unit and ui tests.
 
-The function `tidy_code()` is created in the Rprofile script and therefore is always available in the RStudio console to tidy code according to tidyverse styling using the styler package. This function also helps to test the running of the code and for basic syntax errors such as missing commas and brackets.
+In the `.github` folder there are workflow files that run the automated tests for all pushes to master and PR's. The automated tests cover everything that is run locally in the
```
Comment (lauraselby) [inline README.md]: Is this missing the end of the sentence?

Comment (cjrace) [reply README.md]: I don't think so? Though it does look like it in the changes. Check the readme on the main repo page on the dev branch

File: R/mainTests.r (hunk 1 above)
Comment (cjrace) [reply R/mainTests.r]: How's this? It's getting to be a lot of text 

image

File: README.md (hunk 2 above)
Comment (cjrace) [reply README.md]: Nope, you're right - the merging of master back in probably caused this

File: R/mainTests.r (hunk 1 above)
Comment (lauraselby) [reply R/mainTests.r]: I think that's more informative yeah 👍

**Review by lauraselby — APPROVED** [review]

**Pull request merged**